package.json

```diff
@@ -24,24 +24,29 @@
   "scripts": {
     "test": "asp --verbose",
     "test:ci": "asp --summary",
-    "fmt": "prettier --write '{*,**/*}.{js,ts,jsx,tsx,json}'",
-    "lint:fix": "eslint --fix --ext .ts . && yarn fmt",
-    "lint": "eslint --ext .ts . && prettier --check '{*,**/*}.{js,ts,jsx,tsx,json}'"
+    "prettier": "as-prettier --write '{*,**/*}.{js,ts,jsx,tsx,json}'",
+    "fmt": "as-prettier --write '{*,**/*}.{js,ts,jsx,tsx,json}' && npm run lint:fix",
+    "fmt:check": "as-prettier --check '{*,**/*}.{js,ts,jsx,tsx,json}'",
+    "lint": "eslint --ext .ts .",
+    "lint:fix": "eslint --fix --ext .ts ."
   },
   "devDependencies": {
+    "@as-pect/cli": "^7.0.7",
+    "@assemblyscript/loader": "^0.21.2",
+    "@massalabs/as": "1.0.0",
     "@typescript-eslint/eslint-plugin": "^5.29.0",
     "@typescript-eslint/parser": "^5.29.0",
-    "@as-pect/cli": "^7.0.7",
     "as-console": "^6.0.2",
     "assemblyscript": "^0.21.2",
-    "@assemblyscript/loader": "^0.21.2",
+    "assemblyscript-prettier": "git://github.com/Thykof/assemblyscript-prettier#update-assemblyscript-21",
```

Comment:
I found a betterway to handle the dependency version conflict:
leave ```"assemblyscript-prettier": "^1.0.5",```
just add this in your  package.json
    ```"overrides": {
      "assemblyscript": "^0.21.7"
    }```